cultcargo/builder/build_cargo.py

```diff
@@ -82,7 +84,7 @@ def build_cargo(manifest: str, do_list=False, build=False, push=False, all=False
             "{task.description}",
             console=console, disable=boring) as progress:
         print = progress.console.print
-
+    
```

Comment:
Spurious whitespace. In vscode you can do CTRL+SHIFT+p and type/select `Trim Trailing Whitespace` to have this done for the whole file. Can make for massive diffs though.